spec/models/barcode/format_handlers_spec.rb

```diff
@@ -284,5 +284,15 @@ def self.it_has_an_invalid_barcode(barcode)
     it_has_an_invalid_barcode '00210783400000001 '
   end
 
+  describe Barcode::FormatHandlers::Newcastle do
+    it_has_a_valid_barcode 'ICHNE12345c', prefix: 'ICHNE', number: 12_345, suffix: 'c'
```

Comment:
Why the underscore between the numbers?

Reply:
I think rubocop likes the format. It doesn't make a difference functionally.

Reply:
Oh OK, how random!